src/main.c

```diff
@@ -186,8 +195,15 @@ int main(int argc, char **argv) {
   else
     tauFlag = atoi(argv[3]);
 
-  if (argc > 2)
-    printf("restart %d tau %d\n", restartFlag, tauFlag);
+  if(argc < 5){
+    zerot_flag = 0;
+  } else {
+    zerot_flag = atoi(argv[4]);
+  }
+
+  if (argc > 2){
+    printf("restart %d tau %d zerot %d\n", restartFlag, tauFlag, zerot_flag);
+  }
```

Comment:
What does this zerot flag do?

Reply:
That needs to go --left over from an attempt to restart for from 1d-3v